### PR TITLE
IGH_Issue43_fillMissingness

### PR DESCRIPTION
**Pull request by globusharris**
Resolves issue 43

### CODE REVIEW

File: R/mechanism-gaussian.R [hunk 1]
```diff
@@ -19,18 +19,11 @@ mechanismGaussian$methods(
...
-                x <- fillMissing(x, .self$var.type, .self$impute.rng[1], .self$impute.rng[2])
-            }
-        } else {
-            x <- fillMissing(x, .self$var.type, categories=.self$bins)
-        }
+        x <- fillMissing(x, .self$var.type, impute.rng=.self$rng, categories=.self$bins)
         field.vals <- .self$getFunArgs(fun)
         true.val <- do.call(fun, c(list(x=x), field.vals))
         scale <- sens * sqrt(2 * log(1.25 / .self$delta)) / .self$epsilon
+        noise <- dpNoise(n=length(true.val), scale=scale, dist='gaussian')
```
Comment (tercer) [inline R/mechanism-gaussian.R]: Why is this here?

Comment (globusharris) [reply R/mechanism-gaussian.R]: I must have added that for testing purposes at some point. It should not be there.
